api/applicationserver_pubsub.proto

```diff
@@ -49,7 +49,7 @@ message ApplicationPubSub {
   // The MQTT provider settings.
   message MQTTProvider {
     string server_url = 1 [(gogoproto.customname) = "ServerURL", (validate.rules).string.uri = true];
-    string client_id = 2 [(gogoproto.customname) = "ClientID", (validate.rules).string.max_len = 23];
+    string client_id = 2 [(validate.rules).string.max_len = 23];
```

Comment:
I wanted to scope this to only `identifiers.proto`, but apparently some of our tooling thinks that if we use ClientID anywhere, we're using ClientID everywhere 😕